GVRf/Framework/jni/objects/mesh.cpp

```diff
@@ -180,12 +180,11 @@ void Mesh::getTransformedBoundingBoxInfo(glm::mat4 *Mat,
 void Mesh::generateVAO() {
 #if _GVRF_USE_GLES3_
 
-    if (vaoInitiliased_)
-        return;
+
     GLuint tmpID;
 
-    if (vao_dirty_) {
-         deleteVaos();
+    if (!vao_dirty_) {
+         return;
```

Comment:
I am curious why the deleteVaos() call is no longer necessary in case of a "dirty" mesh?